### PR TITLE
Fix #1: Add description in the README.md

### LINKED ISSUE #1: Add description in the README.md
Currently we don't have description/overview of the app, we should have an overview of the app in README.

### PR DESCRIPTION
**Pull request by deeps0102**
Fixes #1:
Added an overview section in the README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Do-odle
 Multiplayer Pictionary Game 
 
+# Overview of the app
+**Do-odle** is a free multiplayer drawing and guessing game app.It is one of the best friends and family online games for coronavirus quarantine.Each game consists of a few rounds decided by the host.In every round someone has to draw their chosen words and others have to guess the right answer in order to gain points. The faster the player guesses the answer ,more will be the points awarded to that player. The person with the most points at the end of game will then be crowned as the winner!!
```
Comment (prayutsu) [inline README.md]: Please add an space after the full stop
```.... app. It ....```

Comment (prayutsu) [inline README.md]: Please correct it, there should be a space after the comma, not before the comma

**Review by prayutsu — CHANGES_REQUESTED** [review]
@deeps0102 The PR is good, just left a few comments, PTAL(Please take a look)

File: README.md (hunk 1 above)
Comment (deeps0102) [reply README.md]: Done

Comment (deeps0102) [reply README.md]: Done

**Review by prayutsu — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**